src/views/Export.vue

```diff
@@ -2,22 +2,24 @@
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
-import { ParsedSimpleRequest } from '../lib/RequestTypes';
-import { SimpleRequest } from '@nimiq/keyguard-client';
+import { ParsedExportRequest } from '../lib/RequestTypes';
+import { ExportRequest } from '@nimiq/keyguard-client';
```

Comment:
To reduce confusion over request type names, combine this:

```suggestion
import { KeyguardClient } from '@nimiq/keyguard-client';
```

Reply:
I guess without braces?

Reply:
Ah, yes. Importing the namespace, not the property. I was confused because we call the namespace `KeyguardClient` in the other files, but the package also exports a named `KeyguardClient` :smile:.